em/emFluxCorrection.cc

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2011 Scientific Computation Research Center
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+#include <iostream>
```

Comment:
this seems to be not used in this file. In that case, it should be removed.

Reply:
Removed.